### PR TITLE
Additional Riak features with error messages printed

### PR DESCRIPTION
**Pull request by dmyerscough**

### CODE REVIEW

File: salt/modules/riak.py [hunk 1]
```diff
@@ -44,10 +59,20 @@ def stop():
...
+    if cmd['retcode'] != 0:
+        ret['comment'] = cmd['stderr']
+    else:
+        ret['comment'] = cmd['stdout']
+        ret['success'] = True
+
+    return ret
+
+
+def cluster_join(username, hostname):
```
Comment (rallytime) [inline salt/modules/riak.py]: Can you add some `versionchanged` tags here for both of the args that you changed? Can you add some docs for both of these args as well?

File: salt/modules/riak.py [hunk 2]
```diff
@@ -57,12 +82,44 @@ def cluster_join(riak_user=None, riak_host=None):
...
+        ret['comment'] = cmd['stdout']
+        ret['success'] = True
+
+    return ret
+
+
+def cluster_leave(username, hostname):
+    '''
+    Leave a Riak cluster
+
```
Comment (rallytime) [inline salt/modules/riak.py]: `.. versionadded:: Beryllium`

File: salt/modules/riak.py [hunk 3]
```diff
@@ -102,29 +174,54 @@ def member_status():
...
+                'Pending': pending
+            }
+
+    return ret
+
+
+def status():
+    '''
+    Current node status
+
```
Comment (rallytime) [inline salt/modules/riak.py]: `.. versionadded:: Beryllium`

**Comment by rallytime** [comment]
Thanks @dmyerscough! In addition to some of the `versionadded` tags I've mentioned above, I noticed that the actual module doesn't have a versionadded. Since you're fixing up this file and adding some new features, can you add that missing versionadded tag to the top? It looks like this file was added clear back in 0.17.

**Comment by dmyerscough** [comment]
@rallytime I just updated the source with versions added and changed. I refactored the code a bit on nearly all functions.

**Comment by jfindlay** [comment]
@dmyerscough, there are a few [test](https://jenkins.saltstack.com/job/salt-pr-lint-n/6428/violations/file/tests/unit/modules/riak_test.py/) [errors](https://jenkins.saltstack.com/job/salt-pr-rs-ubuntu14.04-n/5991/).

**Comment by dmyerscough** [comment]
@jfindlay I will fix those today

**Pull request merged**